pkg/apis/velero/v1/pod_volume_backup_types.go

```diff
@@ -114,7 +114,6 @@ type PodVolumeBackupStatus struct {
 // +kubebuilder:printcolumn:name="Namespace",type="string",JSONPath=".spec.pod.namespace",description="Namespace of the pod containing the volume to be backed up"
 // +kubebuilder:printcolumn:name="Pod",type="string",JSONPath=".spec.pod.name",description="Name of the pod containing the volume to be backed up"
 // +kubebuilder:printcolumn:name="Volume",type="string",JSONPath=".spec.volume",description="Name of the volume to be backed up"
-// +kubebuilder:printcolumn:name="Repository ID",type="string",JSONPath=".spec.repoIdentifier",description="Backup repository identifier for this backup"
```

Comment:
Shouldn't we just remove the `printcolumn` kubebuilder marker ? And keep the repo ID in the CRD, this way we don't display it in the kubectl get call without actually modifying the PVB CRD.

Reply:
Yes, we didn't change the CRD spec, only removed the marker.

Reply:
ok ok